CHANGELOG_WINBOX.md

```diff
@@ -1,5 +1,23 @@
 # Winbox changelog
 
+## 3.36
+
+- fix issue with non existing records still in winbox ros7
```

Comment:
Please add `;` ad the end of lines and `.` in the end of last line.

Reply:
Done.